.travis.yml

```diff
@@ -1,3 +1,5 @@
+dist: trusty
```

Comment:
Switch from `precise` to `trusty` is required mainly because of `jsdom` dependency. It requires Node.js > 4. Precise has very old Node.js and NPM versions and they can not be updated because of default versions of C++ compilers. While it's possible somehow to install Node 6 on Precise, see this [document](https://github.com/nodesource/distributions/blob/master/OLDER_DISTROS.md), I think this is not elegant solution and I'm not 100% sure it will work correctly with Travis and container-based precise dist.

Trusty has v6.3.0 which is enough and way better and newer.